src/app.jsx

```diff
@@ -73,12 +102,13 @@ class Application extends React.Component {
         this._asyncRequestImages = utils.varlinkCall(utils.PODMAN, "io.podman.ListImages")
```

Comment:
Don't do the ListImages/ListContainers/InspectImage etc. calls here again. Use the utils.js functions for that.